### PR TITLE
Updated styles

### PR DESCRIPTION
**Pull request by yaichenbaum**
- Added a background brush for the file browser control
- Added a background brush for the status bar
- Fixed the margins on the navigation toolbar and status bar

### CODE REVIEW

**Review by winston-de — APPROVED** [review]

**Comment by winston-de** [comment]
Wait, would you mind if I quickly push a change to fix this tiny bit of text cutoff?

image

**Comment by yaichenbaum** [comment]
> Wait, would you mind if I quickly push a change to fix this tiny bit of text cutoff and the drive storage bar alignment?
> 
> image
> image

@winston-de Sure!

**Review by winston-de — APPROVED** [review]

File: Files/UserControls/Widgets/DrivesWidget.xaml [hunk 1]
```diff
@@ -186,7 +186,10 @@
                                     Margin="12,0,0,0"
                                     VerticalAlignment="Center"
                                     Orientation="Vertical">
-                                    <Grid Margin="0,0,0,8" HorizontalAlignment="Stretch">
+                                    <Grid
+                                        Height="30"
```
Comment (yaichenbaum) [inline Files/UserControls/Widgets/DrivesWidget.xaml:190]: Can you try `28`?

Comment (yaichenbaum) [reply Files/UserControls/Widgets/DrivesWidget.xaml:190]: That would be great!

**Pull request merged**